README.md

```diff
@@ -2,7 +2,7 @@
 
 This is a collection of contracts related to specifically to Gnosis Olympia. These contracts can be used in setting up prediction market tournaments.
 
-This is the Olympia token contract implementation:
+This is the pm-apollo-token contract implementation:
```

Comment:
The repo may be changed, but the name of the token contract should still by OlympiaToken iirc. Also, I think these addresses refer specifically to Olympia stuff, so this can remain the same.